### PR TITLE
Feature/nft influencers trx

### PR DESCRIPTION
**Pull request by peratik**
## Changes

- NFT Influencers Trx section added to the home page
- NFT Influencers Trx dashboard pages added

## Notion's card

https://www.notion.so/santiment/NFT-Sanbase-Features-dd9c8b98efef45e0835cb81639f4f839

## Checklist

- [x] I've performed a self-review, followed all rules from [Frontend style guide](https://www.notion.so/santiment/Front-end-style-guide-81750096b38c4bea9a29b14fd4ab8667)
- [x] If I make changes to another person's module, I've asked how to use it or request a review
- [x] I've updated the [documentation](https://github.com/santiment/academy), if necessary (Keyboard shortcuts, Account settings)
- [x] I've checked night mode, mobile & tablet screens (if have changes in UI)
- [x] I've added tests (if necessary)

## Screenshots or GIFs

image

image

image

image

### CODE REVIEW

File: src/pages/Index/Navigation/anchors.js [hunk 1]
```diff
@@ -17,10 +18,26 @@ export const SCROLLABLE_ANCHORS = [
...
 export const TOP_LINKS = [
+  {
+    title: 'NFT Influencers Trx',
+    link: NFTINFLUENCERS_ANCHOR,
+    Icon: () => (
+      <svg className={styles.topIcon} width='32' height='32'>
+        <g opacity='0.9'>
+          <path
+            fill-rule='evenodd'
+            clip-rule='evenodd'
```
Comment (DmitriVanGuard) [inline src/pages/Index/Navigation/anchors.js:34]: `fillRule` and `clipRule`

File: src/pages/Index/Section/NftInfluencers/accounts.json [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+    {
+      "Name": "Pranksy",
+      "Address": "0xd387a6e4e84a6c86bd90c158c6028a58cc8ac459",
+      "ENS": "",
+      "Twitter": "https://twitter.com/Pranksy",
+      "OpenSea": "https://opensea.io/accounts/Pranksy",
+      "Youtube": "",
+      "Notes": "Proof - https://dune.xyz/queries/40614/80127\n&& OpenSea",
+      "Location": "",
+      "Achievment": ""
```
Comment (DmitriVanGuard) [inline src/pages/Index/Section/NftInfluencers/accounts.json:11]: Can it be fetched by using api?

**Review by DmitriVanGuard — APPROVED** [review]

File: src/pages/Index/Section/NftInfluencers/accounts.json (hunk 2 above)
Comment (peratik) [reply src/pages/Index/Section/NftInfluencers/accounts.json:11]: It is not ready, https://discord.com/channels/334289660698427392/483922119873265664/921417515744264262

Comment (Partysun) [reply src/pages/Index/Section/NftInfluencers/accounts.json:11]: True. Can't be yet

**Review by DmitriVanGuard — APPROVED** [review]

**Pull request merged**